src/test/access-control-transformer.test.js

```diff
@@ -0,0 +1,208 @@
+const assert = require('assert');
```

Comment:
Was just looking through the assert docs, seems like it advises the usage of `require('assert').strict` over without it.

Reply:
done